### ISSUE TITLE
Could not post comment.

### ISSUE THREAD

**Issue by redsector72** [InstaPy/InstaPy#2055]
I see this appearing on the bottom line of the Chrome window.
I've checked the log and nothing is in there (Instapy believes to have commented).

The strange thing is that even trying manually to comment, inside chrome (other tab) i get the same.
I've tried commenting the same picture from safari and it went thought.

Digging, I've found that some comments went ok via InstaPy. 
I suppose the majority not.
Maybe they are able to tell it's not human?

I'm not really concerned about commenting, for now I disable this function.

**Comment by uluQulu** [comment]
@redsector72,
I've also encountered it once, and I guess it's a possible result of _excessive commenting_.  
Give that action a bit of rest and it should work fine again.

**Comment by CodeMaster1** [comment]
I have seen this issue as well. Any comments @timgrossmann ?

**Comment by CodeMaster1** [comment]
This behavior gets triggered randomly and even though one comments fails and the other gets through.

**Comment by timojaask** [comment]
I have seen this as well. @uluQulu I don't think it's because of excessive commenting, because I just started running the bot for the first time on this account and I started getting this error on about 5th post. It comes randomly. It's pretty annoying because InstaPy log shows that it made a bunch of comments, when in reality maybe only 10% got through.

The error message says "Couldn't post comment" (putting it here for the search, since the title of the post has it differently)
